File: examples/python/evm/create_account.py
# Usage: uv run python evm/create_account.py

import asyncio

from cdp import CdpClient


async def main():
    async with CdpClient() as cdp:
        account = await cdp.evm.create_account()
        print(f"Successfully created account: {account.address}")


asyncio.run(main())

File: examples/typescript/evm/createAccount.ts
// Usage: pnpm tsx evm/createAccount.ts

import { CdpClient } from "@coinbase/cdp-sdk";
import "dotenv/config";

const cdp = new CdpClient();
const account = await cdp.evm.createAccount();
console.log("Created account: ", account.address);
